#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_include_path.dpatch by Oliver Grassl <O.Grassl@gmx.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Set correct include path to svdrpservice.h.

@DPATCH@
diff -urNad remotetimers-0.1.1~/svdrp.h remotetimers-0.1.1/svdrp.h
--- remotetimers-0.1.1~/svdrp.h	2008-07-14 23:57:23.000000000 +0200
+++ remotetimers-0.1.1/svdrp.h	2009-04-23 13:16:49.000000000 +0200
@@ -9,7 +9,7 @@
 #define _REMOTETIMERS_SVDRP__H
 
 #include <stdlib.h>
-#include "../svdrpservice/svdrpservice.h"
+#include <svdrpservice.h>
 #include <vdr/tools.h>
 #include <vdr/plugin.h>
 
